tests/unit/test_utils.py

```diff
@@ -264,7 +263,9 @@ def test_rmtree_errorhandler_reraises_error(tmpdir: Path) -> None:
             # Tuple[None, None, None]]"; expected "Tuple[Type[BaseException],
             # BaseException, TracebackType]"
             rmtree_errorhandler(
-                mock_func, path, sys.exc_info()  # type: ignore[arg-type]
+                mock_func,
+                subdir_path,
+                sys.exc_info(),  # type: ignore[arg-type]
```

Comment:
Again, I'm not convinced that this doesn't change what's affected by the `# type: ignore` directive.

Reply:
IMO, the correct reformatting here *has* to be

```
                mock_func,  # type: ignore[arg-type]
                subdir_path,  # type: ignore[arg-type]
                sys.exc_info(),  # type: ignore[arg-type]
```

This seems to me like a straight bug in ruff.

Reply:
There appears to be a related discussion on ruff side here: https://github.com/astral-sh/ruff/issues/8286#issuecomment-1807867070

But not specific issue open on ruff side that I can find which exactly matches your concern.

Reply:
I added a note about this case to that issue.